app/validators/participant_profile_active_validator.rb

```diff
@@ -1,11 +0,0 @@
-# frozen_string_literal: true
-
-class ParticipantProfileActiveValidator < ActiveModel::EachValidator
-  def validate_each(record, attribute, value)
-    record.errors.add(attribute, I18n.t("errors.participant_profile.not_active")) unless active?(value)
-  end
-
-  def active?(participant_profile)
-    participant_profile&.active_record? && participant_profile&.training_status_active?
```

Comment:
Im curious as to why this changed, and we no longer want training status active? I've checked our service and we prevent changing things if a participant is withdrawn

Reply:
In the question we made to Nathan about setting training_status: :active when executing the cohort change via this tool we finally concluded to leave training_status as it was originally.
So why required the participants to be training_status: :active?

Also it feels to me that the original training status refers to that training period. Why would that block starting a new training period somewhere else?

In any case, happy to reinstate it if needed

Reply:
having a think maybe we can keep it as it was currently? as I think resetting the training status is quite different than the validation itself. Just erroring on the side of caution as it's already there, if we later find it an issue maybe we can remove it?